django-stubs/db/models/constraints.pyi

```diff
@@ -16,6 +16,7 @@ class Deferrable(Enum):
 class BaseConstraint:
     name: str
     violation_error_message: _StrOrPromise | None
+    default_violation_error_message: str = ...
```

Comment:
We don't usually add `= ...` to class attributes -- that's only for function/method arguments.

```suggestion
    default_violation_error_message: str
```

Reply:
Will keep this in mind, thanks for the review =)